test/deno/npm.test.ts

```diff
@@ -120,6 +120,32 @@ describe("npm for import map", () => {
   }
 });
 
+// NOTE: This feature is not required in the imports field in deno.json and source files.
+// https://github.com/denoland/deno/pull/22087
+// https://deno.com/blog/v1.40#simpler-imports-in-denojson
+test("should accept npm specifier with subpath exports in import map", () => {
```

Comment:
Could you write it using `describe` and `it` instead of `test`?
I would like to keep the format consistent with the others.

Reply:
fix it! 3803deb